HighLevelGoals.md

```diff
@@ -34,5 +34,7 @@
 5. Make a great platform:
     * build a new LLVM backend for WebAssembly and an accompanying
       clang port;
+      ** see the [FAQ] (FAQ.md) why LLVM is being chosen and note it is not the only compiler
```

Comment:
I'd rather downplay this, since it's our front-facing page. Instead I suggest just saying:

> build a new LLVM backend for WebAssembly and an accompanying clang port ([why LLVM first?](FAQ.md#which-compilers-can-i-use-to-build-webassembly-programs));